gapic/schema/metadata.py

```diff
@@ -188,6 +192,28 @@ def python_import(self) -> imp.Import:
                 alias=self.module_alias,
             )
 
+        if self.is_proto_plus_type:
+            # We need to change the import statement to use an
+            # underscore between the module and the version. For example,
+            # change google.cloud.documentai.v1 to google.cloud.documentai_v1.
+            # Check if the package name contains a version.
+            version_regex = "^v\d(alpha|beta)?\d?"
+            regex_match = re.match(version_regex, self.package[-1])
+
+            if regex_match:
```

Comment:
Do we have a test for this import regex rewriting? We should, for at least cases like `v1`, `v10`,`v2beta`,`v20beta``,v20p1`,`v20p1alpha3p5`, don't you think?

Reply:
Done